cristin-commons/src/main/java/no/unit/nva/cristin/common/client/HttpResponseFaker.java

```diff
@@ -19,54 +19,54 @@
 import static no.unit.nva.cristin.model.Constants.REL_PREV;
 import static no.unit.nva.cristin.model.Constants.X_TOTAL_COUNT;
 
-
 @JacocoGenerated
-public class HttpResponseStub implements HttpResponse<String> {
+public class HttpResponseFaker implements HttpResponse<String> {
```

Comment:
I noticed this is class is in main folder. If we wish to share testing logic among projects, we should create a "common-testing" module where we import it as test dependency. We do not mix production and test code.

Reply:
Moved to cristin-testing